spki/src/spki.rs

```diff
@@ -1,11 +1,22 @@
 //! X.509 `SubjectPublicKeyInfo`
 
+#[cfg(feature = "alloc")]
+extern crate base64ct;
+#[cfg(feature = "fingerprint")]
+extern crate sha2;
```

Comment:
You don't need these `extern crate` directives in the 2018 edition. The only one that's needed is linking to liballoc, and that's an artifact of `#![no_std]`

Reply:
Will fix. Cheers

Reply:
Fixed in latest push